### DISCUSSION TITLE
Cursor not available

### DISCUSSION THREAD

**Discussion by balverio1020** [Farfetch/kafkaflow#270] [Q&A]
I have a business need to set the cursor on one of our kafka flow consumers, can anyone tell me how this is done?

**Comment by filipeesch** [comment]
Hi @balverio1020,

Do you mean set the offset?

**Comment by balverio1020** [comment]
yes offset

**Reply by filipeesch** [reply]
You can change the offset using some Consumer methods or you can use the Dashboard to reset or rewind the offset to a point in time. If you give me more context I could help you better :)
